### ISSUE TITLE
Fix `lightness-notation` false negatives for relative color

### ISSUE THREAD

**Issue by jamesnw** [stylelint/stylelint#8986]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  color: lch(from red 1 0 0);
}
```

### What minimal configuration is needed to reproduce the bug?

```js
export default {
  rules: {'lightness-notation': 'percentage'}
};
```

### How did you run Stylelint?

https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilkA2EATvCnmABYAUAZoRALYqExQoCMKADFwJQDcWAL4gANCGoBLPDABySenEQwAHvIAO0gHRgAzjtHgImSQHMEIFWqIAXFLGpIArnlsZsTZzB0lgAcjwSJuTWmF46ALSYENZI1hJGviS+ajCEYDCYMSYwvoJCApgGkMaBAGJE9LHmOtYAntIBmdpGppr0AFb6YrBq+ohuOOggNfUwjdZDJEN4sV4ToliDw3UNEpnhxabhNUiYUEiEUJMo07M1Q0JYIIJAA

### Which Stylelint-related dependencies are you using?

[stylelint](https://stylelint.io/)@17.0.0
 [stylelint-config-standard](https://www.npmjs.com/package/stylelint-config-standard)@40.0.0

### What did you expect to happen?

An error should be reported for a lightness value of `1` when expecting a percentage

### What actually happened?

No problems reported

### Do you have a proposal to fix the bug?

Detect relative color, and shift reporting column

**Comment by jeddy3** [comment]
@jamesnw Thank you for the report and PR! Titling as a false negative. It matters for [semver policy](https://stylelint.io/about/semantic-versioning/).
